testdata/book/cookie.yml

```diff
@@ -0,0 +1,36 @@
+desc: Inherit cookies Example
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    openapi3: https://tryapisproxy.com/spec/httpbin
+vars:
+  cookie:
+    key: "cookie-name"
+    value: "cookie-value"
+debug: ${DEBUG:-true}
+steps:
+  bindUuid:
+    desc: UUIDs are numbered in the image of a session.
+    bind:
+      uuid: faker.UUID()
+  setCookie:
+    desc: Returns a set of response headers from the query string.
+    req:
+      /response-headers?Set-Cookie={{ vars.cookie.key }}%3D{{ uuid }}:
+        get:
+          body: null
+    test: |
+      current.res.status == 200
+      && current.res.cookies[vars.cookie.key].Name == vars.cookie.key
+      && current.res.cookies[vars.cookie.key].Value == uuid
+  inheritCookie:
+    desc: Returns cookie data.
+    req:
+      /cookies:
+        get:
+          headers:
+            Cookie: "{{ cookies[url('${HTTPBIN_END_POINT:-https://httpbin.org/}').Host][vars.cookie.key].Raw }};"
+          body: null
+    test: |
+      current.res.status == 200
+      && current.res.body.cookies[vars.cookie.key] == uuid
```

Comment:
Looks like it needs a new line at the end.

<img width="93" alt="CleanShot 2023-07-17 at 08 13 27@2x" src="https://github.com/k1LoW/runn/assets/57114/05fb8cdc-2750-409b-8b03-40a203770f93">

Reply:
https://github.com/k1LoW/runn/pull/556/commits/c9c5f9e7b695187ffef946d55365f245b85c8cc2